resources/install.sh

```diff
@@ -5,7 +5,7 @@ sed -i s/cGFzc3dvcmQtZ29lcy1oZXJlCg==/$password/g /var/lib/gravity/resources/sec
 
 # Generate password for InfluxDB administrator
 password=$(cat /dev/urandom | tr -dc 'a-zA-Z0-9' | fold -w 32 | head -n 1 | tr -d '\n ' | /opt/bin/base64)
-sed -i s/b1lIV3gyVDlmQVd3SzdsZTRrZDY=/$password/g /var/lib/gravity/resources/secrets.yaml
+sed -i s/b1lIV3gyVDlmQVd3SzdsZTRrZDY=/$password/g /var/lib/gravity/resources/influxdb-secret.yaml
```

Comment:
Where is this file coming from (`influxdb-secret.yaml`)? I don't see it in the repo.